src/views/NotFoundPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import Navigation from "../components/Navigation";
+import "../css/notfoundpage.css";
+
+export default class NotFoundPage extends Component {
+  render() {
+    return (
+      <>
+        <Navigation />
+
+        <div className="container">
+          <div className="row justify-content-center">
+            <div className="col-xs-12 col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3">
+              <div className="row justify-content-center">
+                <h1 className="text-center">Oops! 😭</h1>
```

Comment:
Remove the emoji and possible change the text to something more suitable. Something like "Not Found" or "Can't Find". Currently, it looks too informal.